sentry/src/main/java/io/sentry/ProfilingTraceData.java

```diff
@@ -189,4 +192,12 @@ public boolean isDevice_is_emulator() {
   public void setSampled_profile(@Nullable String sampledProfile) {
     this.sampled_profile = sampledProfile;
   }
+
+  public void readDeviceCpuFrequencies() {
+    try {
+      this.device_cpu_frequencies = deviceCpuFrequenciesReader.call();
+    } catch (Exception e) {
+      this.device_cpu_frequencies = new ArrayList<>();
```

Comment:
Why do we need that? `device_cpu_frequencies` is already created in the field declaration.

Reply:
It's just because I cannot leave exception block empty. Since there's no logger to log the exception i don't know if it's fine to call `e.printStackTrace()`

Reply:
You can, do this:

```
} catch (Throwable ignored) {
      // should never happen
}
```

just name it `ignored` and add a comment.

Reply:
I previously did
```
} catch (Throwable ignored) {
}
```
without the comment and it was complaining. I added the comment and now it's fine, thanks!